### PR TITLE
Add card actions styled components: Waterfall PR[1]

### PR DESCRIPTION
**Pull request by fairlighteth**
WIP part of waterfall PRs

- Adds styled components for card actions (profile page)
Screen Shot 2022-04-05 at 15 36 42
Screen Shot 2022-04-05 at 15 36 30

### CODE REVIEW

File: src/custom/pages/Profile/styled.tsx [hunk 1]
```diff
@@ -429,6 +429,25 @@ export const BannerCard = styled(BannerExplainer)`
   }
 `
 
+export const CardActions = styled.div`
+  width: 100%;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+
+  > a {
```
Comment (nenadV91) [inline src/custom/pages/Profile/styled.tsx:438]: One suggestion, maybe for consistency we can add text color change to orange on `hover` as it works now for the governance card links.

Comment (fairlighteth) [reply src/custom/pages/Profile/styled.tsx:438]: Good find, will update in an upcoming PR 👍🏼

**Comment by nenadV91** [comment]
If you think its not a good idea please ignore this but maybe we could hide the `Add to Metamask` text and increase the icon size on the screens less then `390px` for example so instead of this
Screenshot from 2022-04-06 11-26-54

We get this 
Screenshot from 2022-04-06 11-28-22

**Review by nenadV91 — APPROVED** [review]
Approved with some minor comments.

**Comment by fairlighteth** [comment]
> If you think its not a good idea please ignore this but maybe we could hide the `Add to Metamask` text and increase the icon size on the screens less then `390px` for example so instead of this Screenshot from 2022-04-06 11-26-54
> 
> We get this Screenshot from 2022-04-06 11-28-22

Thank you for reviewing this. In general I'm aiming to use explicit text labels whenever I can. Just seeing a MetaMask icon has a bit of ambiguity in expected action on click. Even though you might say the context 'should make it clear that you're adding an icon to MetaMask'. Not clear perhaps for everyone.

That being said, I was looking to reduce the amount of text needed here by changing
`Add COW to MetaMask`

To simply:
`Add to MetaMask`

In this case I'm inheriting an existing AddToMetamask component. I first need to modify this and make sure it doesn't break functionality everywhere it's important.

Thank you.

**Comment by elena-zh** [comment]
I have not encountered new issues besides the mentioned ones above (and in this comment)

**Comment by fairlighteth** [comment]
Tested and addressed comments. Continuing merging. Feel free to review post-merge.

**Pull request merged**